images/base-app/scripts/launch-comp.sh

```diff
@@ -4,31 +4,30 @@ set -e
 source /opt/gow/bash-lib/utils.sh
 
 function launcher() {
-  APP_TO_LAUNCH=$1
   export GAMESCOPE_WIDTH=${GAMESCOPE_WIDTH:-1920}
   export GAMESCOPE_HEIGHT=${GAMESCOPE_HEIGHT:-1080}
   export GAMESCOPE_REFRESH=${GAMESCOPE_REFRESH:-60}
 
   if [ -n "$RUN_GAMESCOPE" ]; then
-    gow_log "[Gamescope] - Starting: ${APP_TO_LAUNCH}"
+    gow_log "[Gamescope] - Starting: \`$@\`"
 
     GAMESCOPE_MODE=${GAMESCOPE_MODE:-"-b"}
-    /usr/games/gamescope "${GAMESCOPE_MODE}" -W "${GAMESCOPE_WIDTH}" -H "${GAMESCOPE_HEIGHT}" -r "${GAMESCOPE_REFRESH}" -- ${APP_TO_LAUNCH}
+    /usr/games/gamescope "${GAMESCOPE_MODE}" -W "${GAMESCOPE_WIDTH}" -H "${GAMESCOPE_HEIGHT}" -r "${GAMESCOPE_REFRESH}" -- "$@"
   elif [ -n "$RUN_SWAY" ]; then
-    gow_log "[Sway] - Starting: ${APP_TO_LAUNCH}"
+    gow_log "[Sway] - Starting: \`$@\`"
 
     export XDG_SESSION_TYPE=wayland
     mkdir -p $HOME/.config/sway/
     cp /cfg/sway/config $HOME/.config/sway/config
-    # Modify the config file for res and to launch the ${APP_TO_LAUNCH} at the end
+    # Modify the config file for res and to launch the app at the end
     echo "output * resolution ${GAMESCOPE_WIDTH}x${GAMESCOPE_HEIGHT} position 0,0" >> $HOME/.config/sway/config
-    echo "workspace main; exec ${APP_TO_LAUNCH}" >> $HOME/.config/sway/config
+    echo "workspace main; exec $@" >> $HOME/.config/sway/config
```

Comment:
When the lutris closes, sway stays open. I think it might be better here to either terminate sway (`exec $@ && killall sway`) or restart the application (`exec sh -c "while true; do $@; done"`)

Reply:
I see your point but I feel that this might be dependent on the actual app and we can't really take a decision in the `base-app`, for example, one could easily make a full desktop system and would like for Sway to stay up and running no matter how many apps are running there.  
Shall we add a simple env variable like `STOP_SWAY_ON_APP_EXIT`?